packages/evm/src/interpreter.ts

```diff
@@ -234,7 +236,9 @@ export class Interpreter {
         (opCode === 0x56 || opCode === 0x57 || opCode === 0x5e)
       ) {
         // Only run the jump destination analysis if `code` actually contains a JUMP/JUMPI/JUMPSUB opcode
-        this._runState.validJumps = this._getValidJumpDests(this._runState.code)
+        const { jumps, pushs } = this._getValidJumpDests(this._runState.code)
```

Comment:
`pushs` should be named `pushes` so that it's clear that it's the plural form of "push" (gotta love english). Same thing with `cachedPushs`.

Reply:
Have pushed a commit on this.